src/lib/pickles/plonk_curve_ops.ml

```diff
@@ -45,12 +45,13 @@ let add_fast (type f)
   let p3 = (x3, y3) in
   with_label "add_fast" (fun () ->
       assert_
-        { annotation = Some __LOC__
-        ; basic =
-            Kimchi_backend_common.Plonk_constraint_system.Plonk_constraint.T
-              (EC_add_complete
-                 { p1; p2; p3; inf; same_x; slope = s; inf_z; x21_inv } )
-        } ;
+        Snarky_backendless.Constraint.
```

Comment:
Consider moving to a qualifier of `annotation`

Reply:
Done